src/helmholtz/common/helmholtz_config.h

```diff
@@ -71,5 +74,18 @@
   #define s_fabs fabsl
 #endif
 
+inline void zero_derivs2(s_real *grad, s_real *hes){
+  // Set the derivatives to zero
+  if(grad!=NULL){
```

Comment:
Error during compiling = 'NULL' was not declared in this scope

Reply:
This isn't compiling not quite sure why, there are NULLs other places in the code

Reply:
@mengleCMU, give it a shot now.  I didn't test it but I did include stdlib, which should include the definition of NULL.  I restructured a lot of stuff, and it looks like depending on your system things may behave a bit different.

Reply:
This now compiles